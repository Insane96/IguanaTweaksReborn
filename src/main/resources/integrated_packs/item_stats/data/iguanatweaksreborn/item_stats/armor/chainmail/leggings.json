{
	"item": "minecraft:chainmail_leggings",
	"durability": 38,
	"armor": 3.5,
	"modifiers": [

	]
}