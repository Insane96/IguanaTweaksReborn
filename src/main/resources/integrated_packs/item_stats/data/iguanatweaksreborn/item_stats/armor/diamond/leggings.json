{
	"item": "minecraft:diamond_leggings",
	"durability": 180,
	"armor": 4,
	"modifiers": [

	]
}