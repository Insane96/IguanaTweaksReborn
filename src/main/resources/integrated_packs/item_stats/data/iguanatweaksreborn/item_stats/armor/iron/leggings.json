{
	"item": "minecraft:iron_leggings",
	"durability": 105,
	"armor": 2.5,
	"regenerating_absorption": 1.25,
	"modifiers": [

	]
}