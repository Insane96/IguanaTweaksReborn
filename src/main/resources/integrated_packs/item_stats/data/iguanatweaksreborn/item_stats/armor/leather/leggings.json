{
	"item": "minecraft:leather_leggings",
	"durability": 60,
	"armor": 1,
	"regenerating_absorption": 1.75,
	"modifiers": [

	]
}