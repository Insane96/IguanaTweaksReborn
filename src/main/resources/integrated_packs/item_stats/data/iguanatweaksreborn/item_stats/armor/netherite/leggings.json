{
	"item": "minecraft:netherite_leggings",
	"durability": 165,
	"armor": 5,
	"modifiers": [

	]
}