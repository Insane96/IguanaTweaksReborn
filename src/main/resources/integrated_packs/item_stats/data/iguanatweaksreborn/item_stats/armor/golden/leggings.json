{
	"item": "minecraft:golden_leggings",
	"durability": 60,
	"armor": 1,
	"modifiers": [

	]
}